scripts/ci-integration.sh

```diff
@@ -34,8 +35,7 @@ install_kustomize() {
 }
 
 install_kubectl() {
-   wget https://storage.googleapis.com/kubernetes-release/release/v1.10.2/bin/linux/amd64/kubectl \
-     --no-verbose -O /usr/local/bin/kubectl
+   wget https://storage.googleapis.com/kubernetes-release/release/"${KUBECTL_VERSION}"/bin/linux/amd64/kubectl \
```

Comment:
Looks like the second line for the wget shouldn't have been removed (or the `\` should be removed from the previous line)

Reply:
it is a edit error, just fixed, let me monitor the test result